### ISSUE TITLE
Styling is broken for hero page

### ISSUE THREAD

**Issue by yedhink** [bigbinary/wheel#512]
@goutham-subramanyam _a

The styles for the hero page is broken. Please take a look.

I think this PR might be the one that lead to this state: https://github.com/bigbinary/wheel/pull/496

Refer: https://wheel-production.herokuapp.com/

**Comment by goutham-subramanyam** [comment]
<img width="1904" alt="Screenshot 2021-05-21 at 8 34 27 PM" src="https://user-images.githubusercontent.com/24496302/119158554-f3d8a900-ba73-11eb-991d-d8a8b8a76be5.png">

@yedhink _a Can you clarify what you mean by broken?

**Comment by yedhink** [comment]
@goutham-subramanyam

I've attached a screenshot of the UI looks for me.

Note that I haven't scrolled. I can't scroll up either to see the heading `wheel`:
![image](https://user-images.githubusercontent.com/31244999/119158991-6f3a5a80-ba74-11eb-87cd-3325d890176d.png)

I tried fiddling, and setting `min-h-screen` to to top most flex from console makes the UI better but not great, like so:
![image](https://user-images.githubusercontent.com/31244999/119160295-bb39cf00-ba75-11eb-8f3d-3194762b9d64.png)


Any ideas, what might be the issue? Let me know your thoughts.
